restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/AccountsEndpoint.java

```diff
@@ -216,6 +217,13 @@ protected Response getAccount(final String accountSid, final MediaType responseT
             }
         }
 
+        URI baseUri = info.getBaseUri();
+        Organization org = organizationsDao.getOrganization(account.getOrganizationSid());
+        String accountOrgDomain = org.getDomainName();
+        if (!accountOrgDomain.equalsIgnoreCase(baseUri.getHost().toString())) {
```

Comment:
is this validation goign to apply for all endpoints under Account path?

Seems this will only apply to teh GET Account operation, what about the rest of API?